qa-plugin/wysiwyg-editor/qa-plugin.php

```diff
@@ -42,5 +42,6 @@
 
 qa_register_plugin_module('editor', 'qa-wysiwyg-editor.php', 'qa_wysiwyg_editor', 'WYSIWYG Editor');
 qa_register_plugin_module('page', 'qa-wysiwyg-upload.php', 'qa_wysiwyg_upload', 'WYSIWYG Upload');
+qa_register_plugin_phrases('qa-wysiwyg-lang-*.php', 'wysiwyg');
```

Comment:
Should the `'page'` modules be together followed by the `_phrases` or is my TOC kicking again? :)